### PR TITLE
Add handle patch for identifier

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Sessions.Data/Database.fs [hunk 1]
```diff
@@ -48,16 +50,26 @@ let private getTableName (entityType : Type) =
...
+/// Warning: This is not type safe. The value of newValue may be coerced by the database into the column type. 
+/// E.g. Anything can be inserted into a string column. A float will be truncated and inserted into an int column. 
+let updateFieldWhere<'Entity> (propName : string) newValue (filters : Collections.Generic.IDictionary<string,obj>) =
     let entityType = typeof<'Entity>
     let tableName = getTableName entityType
 
-    let updateSql = sprintf "update %s set %s = @%s where id = @id" tableName propName propName
-    let result = getConnection().Execute(updateSql, dict [propName, box newValue
-                                                          "id", box guid])
+    let updateSql = sprintf "update %s set %s = @%s" tableName propName propName
```
Comment (pallenSL) [inline Sessions.Data/Database.fs]: NB. Consistently having a trailing separator is the nicer pattern when assembling stuff like this.

File: Sessions/HandlesController.fs [hunk 2]
```diff
@@ -4,13 +4,18 @@ open System
...
 open System.Web.Http
-
 open HandlesRepository
+open RestModels
 open DataTransform
 
 type HandlesController() =
     inherit ApiController()
 
+    let patch (handleId: int) (op: PatchOp) =
```
Comment (pallenSL) [inline Sessions/HandlesController.fs]: Some (any!) sanity in the method ordering would be nice.  CRUD?  GET PUT POST DELETE?

File: Sessions.Data/Database.fs (hunk 1 above)
Comment (jebbinSCL) [inline Sessions.Data/Database.fs]: wouldn't that mean that the sql comes out with extra spaces?

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**